bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/BridgeType.java

```diff
@@ -86,6 +86,8 @@ public BridgeType(ThingTypeUID uid, List<String> supportedBridgeTypeUIDs, String
      * @param description the human readable description for the according type
      *            (could be null or empty)
      *
+     * @param category provides information about the thing for filtering
+     *
```

Comment:
I see that you didn't start with it, but could you remove all those silly empty lines between the params?